vrltravels/src/test/java/vrltravelspage/vrlhomepage.java

```diff
@@ -1,32 +1,43 @@
 package vrltravelspage;
 
+import static org.testng.Assert.assertEquals;
+
+import java.util.List;
+
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 
 public class vrlhomepage {
 	WebDriver driver;
 	By source=By.xpath("//input[@name='ONWARD_ORIGIN_CITY']");
 	By destination =By.xpath("//input[@name='ONWARD_DESTINATION_CITY']");
 	By journeydate=By.xpath("//input[@name='TRIPSTARTDATE']");
 	By search=By.xpath("//input[@value='Search']");
-	By daytotavel=By.xpath("//tbody/tr[1]/td[3]/a[1]");
+	
 	
 	public vrlhomepage(WebDriver driver) {
 		this.driver=driver;
 	}
-	public void sourcedata(String departureplace) {
-		driver.findElement(source).sendKeys(departureplace);
+	public void journey() {
+		driver.findElement(source).sendKeys("bangalore");
+		driver.findElement(destination).sendKeys("indi");
+		driver.findElement(journeydate).clear();
+		driver.findElement(journeydate).sendKeys("09-12-2021");
+		driver.findElement(search).click();
+		
 	}
-	public void destinationdata(String arrivalplace) {
-		driver.findElement(destination).sendKeys(arrivalplace);
+	public void titileverification() {

```

Comment:
`public void verifyTitle() {
               // Arrange
		String expTitile = "https://www.vrlbus.in/vrl2013/booking_home_page.aspx?q=LWYTJX";
                // Act
		String actualTitle = driver.getTitle();
                // Assert
		assertEquals(actualTitle, expTitile);
	}`